apps/native/app/src/screens/family/family-details.tsx

```diff
@@ -93,45 +139,56 @@ export const FamilyDetailScreen: NavigationFunctionComponent<{
               loading={loading}
               error={!!error}
             />
-            {person?.nationality ? (
+            {'citizenship' in person && person?.citizenship ? (
```

Comment:
Double check. `person?.citizenship` should be sufficient

Reply:
Since I cannot use optional chaining for the type and not all the objects that person can be have the citizenship prop I need to do it like this. Otherwise I get a ts error.